editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/FuseIntegrationProjectCreatorRunnable.java

```diff
@@ -113,6 +113,15 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
 		} catch (CoreException ex) {
 			ProjectTemplatesActivator.pluginLog().logError(ex);
 		}
+		// delete invalid MANIFEST files
+		IResource rs = c.getProject().findMember("src/META-INF/");

```

Comment:
Nope...there is nothing like that in any template. It is generated on the fly and I did not yet figure out where it comes from.